website/server.js

```diff
@@ -0,0 +1,32 @@
+// require express
+var express = require('express');
+
+// parse command line args
+var parseArgs = require('minimist');
+var argv = parseArgs(process.argv.slice(2), opts={
+    "string": "host",
+    "default": {"port": 28070}
```

Comment:
This might limit our dynamic port selection dreams. Let's just drop that idea if you are still tinkering with it and just deal with it as it comes up as we have documentation on it now :)